components/sst_importer/src/config.rs

```diff
@@ -14,6 +14,7 @@ pub struct Config {
     ///
     /// Default is 10m.
     pub import_mode_timeout: ReadableDuration,
+    pub write_global_seqno: bool,
```

Comment:
`SstImporter` is for data import (BR & lightning) and not the right place to have the `write_global_seqno` config. Let's do the following:
* still have the config in `DbConfig` and store the flag with `RocksEngine` struct
* remove `IngestExternalFileOptions::new` method, instead construct the ingestion options in `Engine::ingest_external_file_cf` internal.